code/game/objects/items/implants/implant_zombie.dm

```diff
@@ -0,0 +1,36 @@
+/obj/item/implant/zombie
+	name = "zombie implant"
+	desc = "Fake your death and trick those NT shills"
+	icon_state = "zombie"
+	uses = 2
```

Comment:
```suggestion
	uses = 1
```
Your description says single-use. You have 2 uses. Make up your mind and stick to it.